### ISSUE TITLE
Previously working queries are no longer valid

### ISSUE THREAD

**Issue by rogadev** [DB-db-dron/weatherDB#2]
Courtenay, BC and even Toronto, ON are no longer valid queries and returning the following:
![image](https://user-images.githubusercontent.com/61405141/166113589-116bf6a4-149f-4a5b-81a3-a1c210ecc8f3.png)

These were working yesterday.

**Comment by MasonMags** [comment]
Same problem here. As of this morning, previously functioning queries no longer valid (with any location).

**Comment by change-is-constant** [comment]
@rogadev  and @MasonMags 
I think it may be a temporary issue. Not there anymore. Just like you depend on this API to keep your app working, the API may also depend on some third-party service which may have failed.

**Comment by rogadev** [comment]
Absolutely. Not pointing fingers or blaming anyone. Just wanted to point out the issue. ❤️

**Comment by DB-db-dron** [comment]
Thanks for reporting and letting us know. We are facing some issues with the current hosting. There can be some anomalies sometimes, but this will be addressed as soon as possible. See this comment (https://github.com/DB-db-dron/weatherDB/issues/1#issuecomment-1096923090) for more info.
